elm.json

```diff
@@ -3,7 +3,7 @@
     "name": "mdgriffith/elm-codegen",
     "summary": "A code generation library for Elm",
     "license": "BSD-3-Clause",
-    "version": "4.2.1",
```

Comment:
In general, you should never bump the package version in a PR, unless you're the author